package.json

```diff
@@ -49,7 +49,7 @@
     "@keen.io/icons": "^1.2.1",
     "@keen.io/parser": "^2.1.1",
     "@keen.io/pubsub": "^1.1.0",
-    "@keen.io/query-creator": "^1.0.7",
+    "@keen.io/query-creator": "^1.0.8",
```

Comment:
We need to get rid of not required dependencies - that are right now part of query creator package.